### PR TITLE
change config file content name

### PR DESCRIPTION
**Pull request by InformatiQ**

### CODE REVIEW

File: recipes/config.rb [hunk 1]
```diff
@@ -17,10 +17,10 @@
 # limitations under the License.
 
 file node['qubit_bamboo']['config_path'] do
-  content node['qubit_bamboo']['config'].to_json
+  content node['qubit_bamboo']['json'].to_json
```
Comment (rveznaver) [inline recipes/config.rb]: `node['qubit_bamboo']['json']` doesn't exist.

File: attributes/default.rb [hunk 2]
```diff
@@ -5,44 +5,20 @@
...
+default['qubit_bamboo']['config']['Bamboo']['Endpoint'] = "http://#{node['ipaddress']}:#{node['qubit_bamboo']['config']['port']}"
+default['qubit_bamboo']['config']['Bamboo']['Zookeeper']['Host'] = ['localhost']
+default['qubit_bamboo']['config']['Bamboo']['Zookeeper']['Path'] = '/bamboo'
+default['qubit_bamboo']['config']['Bamboo']['Zookeeper']['ReportingDelay'] = 5
+
+default['qubit_bamboo']['config']['HAProxy']['TemplatePath'] = "#{node['qubit_bamboo']['home']}/haproxy_template.cfg"
+default['qubit_bamboo']['config']['HAProxy']['OutputPath'] = '/etc/haproxy/haproxy.cfg'
+default['qubit_bamboo']['config']['HAProxy']['ReloadCommand'] = 'haproxy -f /etc/haproxy/haproxy.cfg -p /var/run/haproxy.pid -D -sf $(cat /var/run/haproxy.pid)'
+
+default['qubit_bamboo']['StatsD']['Port'] = 8125
```
Comment (rveznaver) [inline attributes/default.rb]: No need for this attribute, you can put the value directly in `default['qubit_bamboo']['config']['StatsD']['Host']`.

File: attributes/default.rb [hunk 3]
```diff
@@ -5,44 +5,20 @@
 
 default['qubit_bamboo']['log_path'] = '/var/log/bamboo-server.log'
 
-default['qubit_bamboo']['config']['marathon_port'] = 8080
-default['qubit_bamboo']['config']['marathon_hosts'] = ["http://localhost:#{node['qubit_bamboo']['config']['marathon_port']}"]
+default['qubit_bamboo']['marathon_port'] = 8080
```
Comment (rveznaver) [inline attributes/default.rb]: No need for this attribute, you can put the value directly in `default['qubit_bamboo']['config']['Marathon']['Endpoint']`.

File: attributes/default.rb [hunk 4]
```diff
@@ -5,44 +5,20 @@
...
+default['qubit_bamboo']['config']['Marathon']['Endpoint'] = ["http://localhost:#{node['qubit_bamboo']['marathon_port']}"].join(',')
```
Comment (rveznaver) [inline attributes/default.rb]: Why the `.join(',')` at the end?

File: attributes/default.rb [hunk 5]
```diff
@@ -5,44 +5,20 @@
...
 
 default['qubit_bamboo']['config']['port'] = 8000
```
Comment (rveznaver) [inline attributes/default.rb]: You will have to rename this value if you don't want it ending up in the generated JSON.

File: attributes/default.rb (hunk 3 above)
Comment (InformatiQ) [inline attributes/default.rb]: this allows rto make it modifiable later

File: attributes/default.rb (hunk 4 above)
Comment (InformatiQ) [inline attributes/default.rb]: it needs to be written in the form of a string containing all the items with a , as a sperator

File: attributes/default.rb (hunk 3 above)
Comment (rveznaver) [inline attributes/default.rb]: It is modifiable either way.

File: attributes/default.rb (hunk 4 above)
Comment (rveznaver) [inline attributes/default.rb]: You are joining a single value array. This is equivalent to writing `"http://localhost:#{node['qubit_bamboo']['marathon_port']}"`.

**Comment by rveznaver** [comment]
+1

**Pull request merged**